locales/en.json

```diff
@@ -188,6 +188,7 @@
     }
   },
   "GetVcModal": {
+    "retrievingId": "Retrieving ID",
```

Comment:
do we need to keep this as "Retrieving ID" or do we need to change to "Retrieving Card"

Reply:
This text is in the "ApplicaionID" screen. Here we can revert UIN/VID through AID which represents IDs. So I   Preferred "Reverting ID"